parsel/csstranslator.py

```diff
@@ -89,6 +101,116 @@ def xpath_text_simple_pseudo_element(self, xpath):
         """Support selecting text nodes using ::text pseudo-element"""
         return XPathExpr.from_xpath(xpath, textnode=True)
 
+    def xpath_first_of_pseudo(self, xpath):
```

Comment:
oh I see, I was understanding that this was kinda equivalent to `nth-of-type`.

So, what's the difference between: `div:nth-of-type(2)` and `div:nth-of(2)` as implemented here?
Or the difference only appears if you have more complex predicates (`div > div:nth-of-type(2)` vs `div > div:nth-of(2)`)?